install/resources/settings.xml

```diff
@@ -1833,6 +1833,15 @@ activity=Order By Last Activity (DESC)]]></optionscode>
 			<settingvalue><![CDATA[username]]></settingvalue>
 			<isdefault>1</isdefault>
 		</setting>
+		<setting name="wolusersperpage">
+			<title>Users Per Page</title>
+			<description><![CDATA[The number of users displayed per page.]]></description>
+			<disporder>4</disporder>
+			<optionscode><![CDATA[numeric
+min=0]]></optionscode>
```

Comment:
What will happen if it is `0`= Maybe the min should be `1`

Reply:
That was initial thought as an alternative to the suggestion by @Starpaul20 above. Doing a bit of hunting through the code for "perpage" though, it seems that it is general practice for there to be a zero minimum for per-page settings which is interpreted as "use the default", which itself seems to be generally (universally?) hard-coded as 20.

So, setting a minimum of 1 is possible and would avoid the need for @Starpaul20's recommended change, but it would be a break with current standard practice.

Reply:
I would agree that if there is a convention used by other settings, we should follow that convention.